native/docs/troubleshooting.md

```diff
@@ -128,7 +131,7 @@ The following error occurs:
 The app is installed correctly on the android emulator but crashes without any error message on start up.
 To solve this issue delete /android/app/build and android/.gradle folder.
 
-## XCode can't find 'your/directory/native/ios/Integreat/de.lproj/InfoPlist.strings'
+## XCode can't find 'bla/bla/de.lproj/InfoPlist.strings'
```

Comment:
Should this really be named like this?

Reply:
I didn't like how long it was before but I can change it back :)